pyproject.toml

```diff
@@ -151,6 +151,9 @@ select = [
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
 
+[tool.uv.workspace]
+members = ["data-raw"]
```

Comment:
I've not seen this pattern before, looks interesting...

Reply:
I _think_ it was autogenerated when I created a package-data specific venv under `data-raw`. Some more information here: https://docs.astral.sh/uv/concepts/projects/workspaces/.

I'm not sure what the best practice is, and haven't found any great resources on the topic of essentially replicating R-package-like data preparation scripts.

Reply:
I was going to suggest an additional environment in the root `pyproject.toml`, but what you have here does seem like what `uv` wants you to do really. I think that the Astral folk are trying to disrupt python packaging a bit atm so I expect that "good practice" may end up being "what is in the `uv` docs" in a couple of years